lib/server/server.js

```diff
@@ -560,6 +560,7 @@ function execute(port) {
     [
       '../docs/**/*', // docs
```

Comment:
Can maybe change this to:

```
'../' + readMetadata.getDocsPath() + '/**/*'
```

or maybe this would be better?

```
join(CWD, '..', readMetadata.getDocsPath(), '**/*')
```

One of those two?

Reply:
Hey @JoelMarcey, 

sure! The first one returns `../docs/doc1.md` while the second returns `'/Users/arturkhusaenov/GitHub/prepack-website/docs/doc1.md'`. But both work fine.

Would you like to add some other files to ignore? Current list of files to be watched is
```
[ '../docs/doc1.md',
  '../docs/doc2.md',
  '../docs/faqs.md',
  'core/',
  'core/Footer.js',
  'i18n/',
  'i18n/en.json',
  'node_modules/',
  'package.json',
  'pages/',
  'pages/en/',
  'pages/en/help.js',
  'pages/en/index.js',
  'pages/en/users.js',
  'sidebars.json',
  'siteConfig.js',
  'static/',
  'static/css/',
  'static/css/custom.css',
  'static/img/',
  'static/img/docusaurus.svg',
  'static/img/favicon.png',
  'static/img/favicon/',
  'static/img/favicon/favicon.ico',
  'static/img/oss_logo.png',
  'yarn.lock' ]
```